client/src/pages/ContestDetails.js

```diff
@@ -124,19 +124,23 @@ const useStyles = makeStyles((theme) => ({
 export default function ContestDetails(props) {
     const classes = useStyles()
     const [activeTab, setActiveTab] = useState(0)
-    const [openDialog, setOpenDialog] = useState(false)
+    const [openDesignDialog, setOpenDialog] = useState(false)
+    const [openInspirationalDialog, setOpenInspirationalDialog] = useState(false)
     const [contest, setContest] = useState(null)
-    const [designOpening, setDesignOpening] = useState({})
+    const [imageOpening, setDesignOpening] = useState({})
```

Comment:
```suggestion
    const [imageOpening, setImageOpening] = useState({})
```

I would change the set variable just so it's similar as the state